app/src/controllers/administrate/projects/sagas.js

```diff
@@ -72,21 +71,25 @@ function* watchSetViewMode() {
   yield takeEvery(START_SET_VIEW_MODE, setViewMode);
 }
 
-function* addProject({ payload: projectName }) {
+function* addProject({ payload: { projectName, organizationSlug } }) {
   try {
-    const { projectKey } = yield call(fetch, URLS.addProject(), {
+    const projectKey = projectName.trim().replace(/\s+/g, '_');
```

Comment:
Looks like we're creating `projectKey` here manually does it will be processed by BE side and override or this `projectKey` will be used permanently in the app. In case of this `projectKey` stay with us permanently should we care about creating new project using only API?

Reply:
It is confuse me to, but we have such acceptance criteria  
![screenshot_07](https://user-images.githubusercontent.com/84175555/196140457-8f2d1cd7-5444-482b-9845-67d53d0c601b.jpg)

Reply:
I guess project key will be prefilled on UI when creating a new project, but user will be able to change it any time.

Reply:
What value we will use to prefill projectKey field?
Do I need to implement this logic right now?

Reply:
Project name.
No, I think it will be in the scope of project creation modal redesign.